portal-web/test/functional/com/liferay/portalweb/paths/MyTestFile.path

```diff
@@ -0,0 +1,52 @@
+<html>
+<head>
+<title>MyTestFile</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyTestFile</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>CLOSE_BUTTON</td>
+	<td>//button[contains(@class,'btn btn-monospaced') and @aria-label='Close']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_OPTION</td>
+	<td>//span[text()='Custom']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>APPLY_CUSTOM_SIZE</td>
+	<td>//button[@aria-label='Apply Custom Size']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>EDIT_HEIGHT</td>
+	<td>//input[contains(@id,'height')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>EDIT_WIDTH</td>
+	<td>//input[contains(@id,'width')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SIMULATION_BUTTON</td>
+	<td>//button[@data-qa-id='simulation']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSERT_VALUES</td>
+	<td>//div[text()='Custom Size 1080x1920 Applied']</td>
```

Comment:
Next time use a variable to set the values, that way you will make the path reusable.
Ex: //div[text()='Custom Size ${customSizeValue} Applied']